html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Welcome to my Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="logo-container">
+      <a class="text-logo" href="#">Yassine</a>
+    </div>
+    <nav class="nav-menu">
+      <img src="/images/Union.png" alt="menu icon">
+    </nav>
+  </header>
+  <main class="headline">
+      <div class="titel-container">
+        <h2 class="main-titel">Hey there. I’m Yassine </h2>

```

Comment:
- [ ] Remember that if you use `<hX>` tags, you should have an `<h1>` tag on your page. You could use it here instead of the `<h2>` tag because it's the most important title for this page :raised_hands: